src/GeneralStateTestsFiller/stEIP1153/21_tstoreCannotBeDosdOOOFiller.yml

```diff
@@ -0,0 +1,105 @@
+# 21
+# Similar to 15. Transient storage cannot be DOS’d: CALL(self, ...), TSTORE(x, y), TSTORE(x + 1, y), TSTORE(x + 2, y), ... ,TSTORE(x + n, y), REVERT, TLOAD(x) returns 0 and runs in under 15 seconds
+# tstore in a non-monotonic order skipping some slots.
+#
+# Expect slot 0 to have 0. Call fails.
+# Expect slot 1 to have 0. Loading returns 0.
+
+21_tstoreCannotBeDosdOOO:
+  _info:
+    comment: Transient storage cannot be DOS’d.
+
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: 0x20000
+    currentNumber: 1
+    currentTimestamp: 1000
+    currentGasLimit: 0x10000000000000
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+    currentBaseFee: 10
+
+  pre:
+    000000000000000000000000000000000000000A:
+      balance: 1000000000000000000
+      nonce: 0
+      code: |
+        :yul {
+          switch selector()
+
+          case 0x883264e8 { // doCall(uint)
+            let n:= div(calldataload(32), 0x100000000000000000000000000000000000000000000000000000000)
+            doCall(n)
+          }
+
+          case 0xf7d6c0eb { // doNStores(n)
+            let n := calldataload(32)
+            doNStores(n)
+          }
+
+          default {
+            sstore(0, 5)
+          }
+          
+          function doCall(n) {
+            mstore(0, hex"f7d6c0eb") // doNStores(n)
+            mstore(32, n)
+            let t := call(gas(), 0x000000000000000000000000000000000000000A, 0, 0, 64, 0, 0)
+            sstore(0, t) // should revert
+
+            let v := verbatim_1i_1o(hex"b3", 0)
+            sstore(1, v)
+          }
+
+          function doNStores(n) {
+            let val := exp(2, 256)
+            for { let i := 0 } lt(i, n) { i := add(i, 1) } {
+              verbatim_2i_0o(hex"b4", i, 600)
+              let even := mod(i, 2)
+              let index
+              switch even 
+              case 0 {
+                index := i
+                verbatim_2i_0o(hex"b4", i, 600)
+              }
+              case 1 {
+                index := sub(val, i)
+                 verbatim_2i_0o(hex"b4", index, 600)
+               }
+              default {}
```

Comment:
no default needed

Reply:
addressed all the above https://github.com/ethereum/tests/pull/1091/commits/bc0dcbc292b8256d9566b05be2eccab60a2e73c9 &
https://github.com/ethereum/tests/pull/1091/commits/eb0a02818c20be023e6407928054d3478b51b899 & 
https://github.com/ethereum/tests/pull/1091/commits/6a5435dace7150661d5a5bba33840181511d2314